src/config.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/prefer-default-export */
+
+// Target protocol schema version. Used to determine compatibility & migration
+export const APP_SCHEMA_VERSION = 2;
+
+export const APP_SUPPORTED_SCHEMA_VERSIONS = ['1.0.0', 1, 2];
```

Comment:
Does this make sense now that there is a plan to auto-migrate? e.g. if there is a migration path, it's supported, otherwise it isn't?

Reply:
Discussed on slack, and resolved 👍